atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/KvsMigrationCommand.java

```diff
@@ -150,21 +152,29 @@ private AtlasDbConfig makeOfflineIfNecessary(AtlasDbConfig atlasDbConfig) {
     }
 
     public AtlasDbServices connectFromServices() throws IOException {
-        AtlasDbConfig fromConfig = AtlasDbConfigs.load(fromConfigFile, configRoot, AtlasDbConfig.class);
+        AtlasDbConfig fromConfig = overrideTransactionTimeoutMillis(
+                AtlasDbConfigs.load(fromConfigFile, configRoot, AtlasDbConfig.class));
         ServicesConfigModule scm = ServicesConfigModule.create(
                 makeOfflineIfNecessary(fromConfig), AtlasDbRuntimeConfig.withSweepDisabled());
         return DaggerAtlasDbServices.builder().servicesConfigModule(scm).build();
     }
 
     public AtlasDbServices connectToServices() throws IOException {
-        AtlasDbConfig toConfig = toConfigFile != null
-                ? AtlasDbConfigs.load(toConfigFile, configRoot, AtlasDbConfig.class)
-                : AtlasDbConfigs.loadFromString(inlineConfig, null, AtlasDbConfig.class);
+        AtlasDbConfig toConfig = overrideTransactionTimeoutMillis(
+                toConfigFile != null
+                        ? AtlasDbConfigs.load(toConfigFile, configRoot, AtlasDbConfig.class)
+                        : AtlasDbConfigs.loadFromString(inlineConfig, null, AtlasDbConfig.class));
         ServicesConfigModule scm = ServicesConfigModule.create(
                 makeOfflineIfNecessary(toConfig), AtlasDbRuntimeConfig.withSweepDisabled());
         return DaggerAtlasDbServices.builder().servicesConfigModule(scm).build();
     }
 
+    private AtlasDbConfig overrideTransactionTimeoutMillis(AtlasDbConfig config) {
+        return ImmutableAtlasDbConfig.builder().from(config)
+                .transactionReadTimeoutMillis(TRANSACTION_READ_TIMEOUT_MILLIS_OVERRIDE)
```

Comment:
If we're going to do it this way, we should probably also disable retries in the validator; if you hit a problem after a couple hours it doesn't really buy you much to retry, becuase nobody will sit there for that long.